### ISSUE TITLE
Script to download an Amazon S3 Bucket

### ISSUE THREAD

**Issue by HarshCasper** [HarshCasper/Rotten-Scripts#75]
## AIM

Create a Script that can download an Amazon Web Services (AWS) S3 Bucket.

## Languages

- BASH
- Python
- Javascript

## Details

Create a Script that can take the AWS Credentials and download the content of the specified S3 Bucket onto the Local Machine.

**Comment by HeroicHitesh** [comment]
@HarshCasper I am interested in working on this issue, assign it to me. And are we supposed to use `boto3` to solve this issue or is there any other way?

**Comment by HarshCasper** [comment]
@HeroicHitesh I have assigned you the Issue. You can make use of `boto3` to solve this Issue.

**Comment by sameersrivastava13** [comment]
Can i work on this issue? @HarshCasper

**Comment by HeroicHitesh** [comment]
@HarshCasper, sorry for the delayed response, unfortunately, there's no way to download the entire bucket at once using `boto3` as far as I read. So, finally, I decided to create a Python script without using any SDKs by utilizing `AWS CLI` directly. I am done with the script waiting for approval of #200, as soon as it gets merged I'll make a PR to resolve this issue also.

Sorry, @sameersrivastava13 but I have created the script for this issue.
